vendor/brave-ios/Ads/BATBraveAds.h

```diff
@@ -7,20 +7,38 @@
 #import "ads.mojom.objc.h"
 
 typedef NS_ENUM(NSInteger, BATAdNotificationEventType) {
-  BATAdNotificationEventTypeViewed,       // = ads::AdNotificationEventType::kViewed
-  BATAdNotificationEventTypeClicked,      // = ads::AdNotificationEventType::kClicked
-  BATAdNotificationEventTypeDismissed,    // = ads::AdNotificationEventType::kDismissed
-  BATAdNotificationEventTypeTimedOut      // = ads::AdNotificationEventType::kTimedOut
+  BATAdNotificationEventTypeServed,   // = ads::AdNotificationEventType::kServed
+  BATAdNotificationEventTypeViewed,   // = ads::AdNotificationEventType::kViewed
+  BATAdNotificationEventTypeClicked,  // =
+                                      // ads::AdNotificationEventType::kClicked
+  BATAdNotificationEventTypeDismissed,  // =
+                                        // ads::AdNotificationEventType::kDismissed
+  BATAdNotificationEventTypeTimedOut  // =
+                                      // ads::AdNotificationEventType::kTimedOut
 } NS_SWIFT_NAME(AdNotificationEventType);
 
+typedef NS_ENUM(NSInteger, BATInlineContentAdEventType) {
+  BATInlineContentAdEventTypeServed,  // =
+                                      // ads::InlineContentAdEventType::kServed
+  BATInlineContentAdEventTypeViewed,  // =
+                                      // ads::InlineContentAdEventType::kViewed
+  BATInlineContentAdEventTypeClicked  // =
+                                      // ads::InlineContentAdEventType::kClicked
+} NS_SWIFT_NAME(InlineContentAdEventType);
+
 typedef NS_ENUM(NSInteger, BATNewTabPageAdEventType) {
-  BATNewTabPageAdEventTypeViewed,         // = ads::NewTabPageAdEventType::kViewed
-  BATNewTabPageAdEventTypeClicked         // = ads::NewTabPageAdEventType::kClicked
+  BATNewTabPageAdEventTypeServed,  // = ads::NewTabPageAdEventType::kServed
+  BATNewTabPageAdEventTypeViewed,  // = ads::NewTabPageAdEventType::kViewed
+  BATNewTabPageAdEventTypeClicked  // = ads::NewTabPageAdEventType::kClicked
 } NS_SWIFT_NAME(NewTabPageAdEventType);
 
 typedef NS_ENUM(NSInteger, BATPromotedContentAdEventType) {
-  BATPromotedContentAdEventTypeViewed,    // = ads::PromotedContentAdEventType::kViewed
-  BATPromotedContentAdEventTypeClicked    // = ads::PromotedContentAdEventType::kClicked
+  BATPromotedContentAdEventTypeServed,  // =
+                                        // ads::PromotedContentAdEventType::kServed
+  BATPromotedContentAdEventTypeViewed,  // =
+                                        // ads::PromotedContentAdEventType::kViewed
+  BATPromotedContentAdEventTypeClicked  // =
+                                        // ads::PromotedContentAdEventType::kClicked
 } NS_SWIFT_NAME(PromotedContentAdEventType);
```

Comment:
Didn't realize these already existed in ads mojom (not sure when that happened…?), which means we shouldn't be defining these ourselves, and should just use mojom generated enums. Please delete these definitions and refactor other definitions to use their mojom generated Obj-C counterpart:

`BATAdNotificationEventType` → `BATBraveAdsAdNotificationEventType`
`BATInlineContentAdEventType` → `BATBraveAdsInlineContentAdEventType`
`BATNewTabPageAdEventType` → `BATBraveAdsNewTabPageAdEventType`
`BATPromotedContentAdEventType` → `BATBraveAdsPromotedContentAdEventType`

Reply:
Fixed